backend/de.metas.business/src/main/java/de/metas/order/model/interceptor/C_Order.java

```diff
@@ -225,18 +225,21 @@ public void setIncoterms(final I_C_Order order)
 			return; // nothing to do yet
 		}
 
-		final int c_Incoterms;
+		final int incotermsId;
 
 		if (order.isSOTrx())
 		{
-			c_Incoterms = bpartner.getC_Incoterms_Customer_ID();
+			incotermsId = bpartner.getC_Incoterms_Customer_ID();
 		}
 		else
 		{
-			c_Incoterms = bpartner.getC_Incoterms_Vendor_ID();
+			incotermsId = bpartner.getC_Incoterms_Vendor_ID();
 		}
 
-		order.setC_Incoterms_ID(c_Incoterms);
+		if (incotermsId > 0)
```

Comment:
the dev-test sais

> - Clone an order with Incoterms set
>    - the new order has the incoterm copied

....but from this snippet here, i can easily construct a test-case that still fails
- create a sales-order with incoterm A
- set the bpartner's `C_Incoterms_Customer_ID` to incoterm B
- clone the sales order
- the clone has incoterm B ❗